salt/modules/mac_power.py

```diff
@@ -122,30 +122,20 @@ def _validate_enabled(enabled):
     '''
     Helper function to validate the enabled parameter. Boolean values are
     converted to "on" and "off". String values are checked to make sure they are
-    either "on" or "off". All other values return an error.
+    either "on" or "off". Int 1+ and 0 are converted to "on" and "off"
 
     Returns: "on" or "off" or errors
     '''
-    if isinstance(enabled, bool):
-        if enabled:
-            return 'on'
+    if isinstance(enabled, str):
+        if enabled.lower() in ['on', 'off']:
+            return enabled.lower
```

Comment:
Oh, I see it now. enabled.lower needs the () at the end. It must never reach this logic because on and off get turned into True and False by the CLI.